ordered logistic regression.ipynb

```diff
@@ -16,34 +16,50 @@
    "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">num_samples=len(df)//100,
</pre><p>why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MarcoGorelli/pydataglobal-21/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Just like any other splitting size. I had divided it into 1 percent of the dataset size.

Reply:
let's keep this to 1000 (both warmup and num_samples) so the posterior is a more representative

Reply:
OK.

Reply:
OK.